src/globus_cli/exception_handling/hooks/generic_hooks.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import json
+
+import click
+import globus_sdk
+
+from globus_cli.termio import PrintableErrorField, outformat_is_json, write_error_info
+
+from ..registry import sdk_error_handler
+
+
+@sdk_error_handler(
+    error_class="GlobusAPIError", condition=lambda err: err.raw_json is None
+)
+def null_data_error_handler(exception: globus_sdk.GlobusAPIError) -> None:
+    write_error_info(
+        "GlobusAPINullDataError",
+        [PrintableErrorField("error_type", exception.__class__.__name__)],
+    )
+
+
+@sdk_error_handler(
+    error_class="GlobusAPIError", condition=lambda err: outformat_is_json()
+)
+def json_error_handler(exception: globus_sdk.GlobusAPIError) -> None:
+    msg = json.dumps(exception.raw_json, indent=2, separators=(",", ": "))
```

Comment:
We no longer need `separators` specified anywhere. The default changed in Python 3.4 to match what's written here and elsewhere.

```suggestion
    msg = json.dumps(exception.raw_json, indent=2)
```

Reply:
Showing my age here, I guess! I had completely missed that change when it went out. I'm so happy to not have to keep doing this (pointlessly, it turns out!).

Thanks!

Reply:
Yep! I couldn't imagine that this was necessary but had to check the docs, which more-or-less say "changed in Python 3.4 so the defaults are sane". :laughing: